hack/make-rules/test-cmd-util.sh

```diff
@@ -512,6 +512,11 @@ run_pod_tests() {
   ### Fail creating a pod disruption budget if both maxUnavailable and minAvailable specified
   ! kubectl create pdb test-pdb --selector=app=rails --min-available=2 --max-unavailable=3 --namespace=test-kubectl-describe-pod
 
+  ### Create a daemonset
+  kubectl create daemonset test-ds --image=test-image --namespace=test-kubectl-describe-pod
+  # Post-condition: daemonset exists and has expected values
+  kube::test::get_object_assert 'daemonset/test-ds --namespace=test-kubectl-describe-pod' "{{$id_field}}" 'test-ds'
```

Comment:
Please move this test to `run_daemonset_tests()`, and write `kubectl create daemonset` tests similar to: 
https://github.com/kubernetes/kubernetes/blob/4425841ce2a99dfc20f15d2e4813b635520ef19d/hack/make-rules/test-cmd-util.sh#L2691-L2723